Model/Order/Guest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace TNW\Stripe\Model\Order;
+
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Sales\Model\ResourceModel\Order\CollectionFactory;
+use Magento\Sales\Model\ResourceModel\Order\Collection;
+use TNW\Stripe\Helper\Customer as CustomerHelper;
+use TNW\Stripe\Model\Adapter\StripeAdapterFactory;
+use TNW\Stripe\Helper\Payment\Formatter;
+/**
+ * Class Guest
+ * @package TNW\Stripe\Model\Order
+ */
+class Guest
+{
+    use Formatter;
+
+    /**
+     * @var CollectionFactory
+     */
+    protected $orderCollectionFactory;
+
+    /**
+     * @var StripeAdapterFactory
+     */
+    private $adapterFactory;
+
+    /**
+     * @var CustomerHelper
+     */
+    protected $customerHelper;
+
+    /**
+     * Guest constructor.
+     * @param CollectionFactory $collectionFactory
+     * @param StripeAdapterFactory $adapterFactory
+     * @param CustomerHelper $customerHelper
+     */
+    public function __construct(
+        CollectionFactory $collectionFactory,
+        StripeAdapterFactory $adapterFactory,
+        CustomerHelper $customerHelper
+    ) {
+        $this->orderCollectionFactory = $collectionFactory;
+        $this->adapterFactory = $adapterFactory;
+        $this->customerHelper = $customerHelper;
+    }
+
+    /**
+     * @return Collection
+     */
+    public function getGuestOrders()
+    {
+        /**
+         * @var Collection $orderCollecion
+         */
+        $orderCollecion = $this->orderCollectionFactory
+            ->create()
+            ->addFieldToSelect('*');
+
+        $orderCollecion
```

Comment:
put those addAttributeToFilter into above construction

Reply:
there is no need in addition variable here just:
`


return $this->orderCollectionFactory
->create()
 ->addFieldToSelect('*')
->addAttributeToFilter('customer_is_guest', ['eq' => 1])
->addAttributeToFilter('guest_order_exported', ['null' => null]);
`